crates/biome_cli/src/commands/rage.rs

```diff
@@ -226,6 +253,17 @@ where
     }
 }
 
+struct DebugDisplayJson<T>(T);
+
+impl<T> Display for DebugDisplayJson<T>
+where
+    T: Serialize,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> io::Result<()> {
+        write!(f, "{}", serde_json::to_string(&self.0).unwrap())
```

Comment:
I think it can be more safe.
```suggestion
        let str = serde_json::to_string(&self.0)
            .map_err(|_| io::Error::new(io::ErrorKind::Other, "Failed to serialize"))?;
        write!(f, "{}", str)
```